src/components/Modal/Modal.vue

```diff
@@ -728,20 +733,22 @@ $header-size: 50px;
 	&--small {
 		.modal-container {
 			width: 350px;
+			max-height: 90%;
 			height: 450px;
 		}
 	}
 	&--medium {
 		.modal-container {
 			width: 500px;
+			max-height: 90%;
 			height: 600px;
 		}
 	}
 	&--normal {
 		.modal-container {
-			max-width: 70%;
+			max-width: 90%;
```

Comment:
Why changing the size? Only aesthetics?

Reply:
see https://github.com/nextcloud/nextcloud-vue/pull/2197#discussion_r699447477